### ISSUE TITLE
VCF specifications: GQ of type Integer

### ISSUE THREAD

**Issue by wdecoster** [pjedge/longshot#47]
Hi,

The VCF specification says that FORMAT/GQ should be an integer, and not a float as outputted by LongShot. While bcftools doesn't complain about it, pysam does throw an error when parsing the file...

```
[E::vcf_parse_format] Invalid character '.' in 'GQ' FORMAT field at chr10:38970
Traceback (most recent call last):
(...)
  File "pysam/libcbcf.pyx", line 4108, in pysam.libcbcf.VariantFile.__next__
OSError: unable to parse next record
```

I can round these now to work around this issue, so no real trouble. I just wanted to inform you.

Cheers,
Wouter

**Comment by vibansal** [comment]
Thanks for reporting this. It should be fixed soon.

**Comment by vibansal** [comment]
Took a long time, but finally fixed (master branch).

**Comment by wdecoster** [comment]
Thanks!

**Comment by bkehr** [comment]
Hi,

I have a some more things related to this that should be easy to fix:

1) The VCF header now specifies the GQ as Integer but the values are still floating point numbers. As a consequence, `bcftools` complains about the "." in the GQ fields.
2) `bcftools` complains about the DP field not being specified in the header. Adding the following line fixes it:
```
##FORMAT=<ID=DP,Number=1,Type=Integer,Description="Read depth">
```

Thanks,
Birte

**Comment by vibansal** [comment]
These issues have been fixed in a new release (0.4.2). Thanks.
